src/app/modules/shared/services/constraint.service.ts

```diff
@@ -25,14 +31,20 @@ export class ConstraintService {
   }
 
   updatePatients() {
+    this.loadingStateInclusion = "loading";
```

Comment:
The problem is that the string values are also used for the animation state. But I now use a string literal type (LoadingState), to prevent it from having an invalid value.